### ISSUE TITLE
Error on valid syntax

### ISSUE THREAD

**Issue by rudy-lath-vizio** [prettier/plugin-xml#742]
Example:
`<Text text="< Back"  />`

This currently fails with `SyntaxError: unexpected character: ->"<- at offset: XXX skipped 1 characters. (XXX:XXX)` and doesn't format that file at all. I think the issue is with the `<` inside of the quotes. This is not invalid syntax. Can we please have this bug fixed?

**Comment by kddnewton** [comment]
It is specifically called out in the spec as invalid syntax. Here is the link: https://www.w3.org/TR/xml/#CleanAttrVals. It is not a bug. The parser we use does not recover from these types of errors because they're ambiguous.

**Comment by rudy-lath-vizio** [comment]
@kddnewton Is it possible to add a flag that skips that line and formats the rest of the file?

**Comment by kddnewton** [comment]
I don't have control over it since we're using a 3rd-party parser. It's here: https://github.com/SAP/xml-tools. You're welcome to ask them, if they support such a flag I'll add it to prettier

**Comment by rudy-lath-vizio** [comment]
Awesome, I will try and reach out to them. Thanks!

**Comment by rudy-lath-vizio** [comment]
Yes I am doing that right now, I can try to add you to that conversation. And I will also delete my prettier issue, I thought that was the right place to request CLI features.
